vms/event/models.py

```diff
@@ -12,6 +12,12 @@ class Event(models.Model):
                 r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\.)|(,)|(\-)|(!)|(\')]+$', ),
         ],
         )
+    description = models.TextField(
+        blank=True,
+        validators=[
+            RegexValidator(r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\.)|(,)|(\-)|(!)]+$', ),
```

Comment:
Does this define all the possibilities for description field?
I think it limits a few rather.

Reply:
@kamsuri I guess I need to remove these validations, since in portal there are not any.